docs/reference/search/aggregations/bucket/sampler-aggregation.asciidoc

```diff
@@ -0,0 +1,159 @@
+[[search-aggregations-bucket-sampler-aggregation]]
+=== Sampler Aggregation
+
+A filtering aggregation used to limit any nested aggregations' processing to a sample of the top-scoring documents.
```

Comment:
Can we say sub-aggregation here so we don't overload 'nested aggregation' which could cause confusion?